hips/draw/simple.py

```diff
@@ -391,16 +391,16 @@ def make_sky_image(geometry: WCSGeometry, hips_survey: HipsSurveyProperties, til
     ----------
     geometry : `~hips.utils.WCSGeometry`
         Geometry of the output image
-    hips_survey : str or `~hips.HipsSurveyProperties`
+    hips_survey : `str` or `~hips.HipsSurveyProperties`
         HiPS survey properties
     tile_format : {'fits', 'jpg', 'png'}
         Format of HiPS tile to use
         (some surveys are available in several formats, so this extra argument is needed)
 
     Returns
     -------
-    image : `~numpy.ndarray`
-        Output image pixels
+    result : `~hips.HipsDrawResult`
+        HipsDrawResult object
```

Comment:
Don't duplicate the type name in the description. It's not useful. Here I'd suggest this desription:
```
    Result object
```